src/wiki/development/build-instructions.md

```diff
@@ -134,16 +134,60 @@ The path to the .rpm packages will be printed once the build is complete.
 
 ### Building a Flatpak
 
-You don't need to clone the entire Prism Launcher repo for this; the Flatpak file handles that.
+You don't need to clone the entire Prism Launcher repo for the latest stable; the Flatpak file handles that. However, editing of the file is required if you desire to build from the latest commit (cloning the source repo).
 Both `flatpak` and `flatpak-builder` must be installed on your system to proceed.
 
+#### Latest Stable
+
 ```bash
-git clone https://github.com/flathub/org.prismlauncher.PrismLauncher
+git clone --recursive https://github.com/flathub/org.prismlauncher.PrismLauncher
 cd org.prismlauncher.PrismLauncher
 # remove --user --install if you want to build without installing
 flatpak-builder --user --install flatbuild org.prismlauncher.PrismLauncher.yml
 ```
 
+#### Latest Commit
```

Comment:
i think it might be better to suggest using the [upstream](https://github.com/PrismLauncher/PrismLauncher/blob/develop/flatpak/org.prismlauncher.PrismLauncher.yml) manifest. it wouldn't require editing and would have any new additions not present in the stable version that are required for current develop (there aren't right now afaik, but it's better to be safe :p)

Reply:
alrighty